assets/common.sh

```diff
@@ -13,8 +13,18 @@ start_docker() {
       mount -n -t cgroup -o $d cgroup /sys/fs/cgroup/$d
   done
 
+  for i in $(seq 0 100) ; do
+    mknod -m 660 /dev/loop${i} b 7 ${i}
+  done
+
   mkdir -p /var/lib/docker
-  mount -t tmpfs -o size=100G none /var/lib/docker
+  image=$(mktemp $PWD/docker.img.XXXXXXXX)
+  dd if=/dev/zero of=${image} bs=1 count=0 seek=100G
+  mkfs.ext4 -F ${image}
+  losetup -f ${image}
```

Comment:
Could this be changed to `losetup -f --show` instead of grepping for the image? That'll atomically acquire and print out which loopback device was picked. Hopefully that flag's available in the image, not sure. With that change you also no longer need the randomly generated image name.